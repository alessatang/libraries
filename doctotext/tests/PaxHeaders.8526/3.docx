30 atime=1389151468.849904908
30 ctime=1389151468.849904908
