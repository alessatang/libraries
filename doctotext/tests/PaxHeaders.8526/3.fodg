30 mtime=1384978086.751497441
30 atime=1389151468.850904904
30 ctime=1389151468.850904904
