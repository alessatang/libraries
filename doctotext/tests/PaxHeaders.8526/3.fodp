30 mtime=1384978086.761497444
30 atime=1389151469.335902945
30 ctime=1389151469.335902945
