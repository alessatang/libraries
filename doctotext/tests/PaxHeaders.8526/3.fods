30 mtime=1384978086.763497445
30 atime=1389151469.336902941
30 ctime=1389151469.336902941
