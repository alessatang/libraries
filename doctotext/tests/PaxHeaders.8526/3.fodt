30 mtime=1384978086.764497445
30 atime=1389151469.336902941
30 ctime=1389151469.336902941
