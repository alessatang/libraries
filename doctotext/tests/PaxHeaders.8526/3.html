30 atime=1389151469.336902941
30 ctime=1389151469.336902941
