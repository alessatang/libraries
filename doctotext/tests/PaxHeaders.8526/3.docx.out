30 mtime=1379553520.998580287
30 atime=1389151468.850904904
30 ctime=1389151468.850904904
